cmake/Modules/CocosBuildSet.cmake

```diff
@@ -24,8 +24,8 @@ message(STATUS "ENGINE_BINARY_PATH:" ${ENGINE_BINARY_PATH})
 
 # the default behavior of build module
 option(DEBUG_MODE "Debug or Release?" ON)
-option(BUILD_LUA_LIBS "Build lua libraries" OFF)
-option(BUILD_JS_LIBS "Build js libraries" OFF)
+option(BUILD_LUA_LIBS "Build lua libraries" ON)
+option(BUILD_JS_LIBS "Build js libraries" ON)
```

Comment:
Other platforms turn it on by default? Why not use the configuration?

Reply:
We need to use luatests and jstests to test the functionality of the engine, but the default value is off, making each modification more cumbersome. Therefore, we choose to keep this switch on. The next update of PR will close it